pkg/kubectl-argo-rollouts/cmd/status/status.go

```diff
@@ -0,0 +1,122 @@
+package status
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/argoproj/argo-rollouts/pkg/kubectl-argo-rollouts/info"
+	"github.com/argoproj/argo-rollouts/pkg/kubectl-argo-rollouts/options"
+	"github.com/argoproj/argo-rollouts/pkg/kubectl-argo-rollouts/viewcontroller"
+	"github.com/spf13/cobra"
+)
+
+const (
+	statusLong = `Watch rollout until it finishes or the timeout is exceeded. Returns success if
+the rollout is healthy upon completion and an error otherwise.`
+	statusExample = `
+	# Watch the rollout until it succeeds
+	%[1]s status gestbook
+
+	# Watch the rollout until it succeeds, fail if it takes more than 60 seconds
+	%[1]s status --timeout 60 guestbook
+	`
+)
+
+type StatusOptions struct {
+	Watch   bool
+	Timeout int64
+
+	options.ArgoRolloutsOptions
+}
+
+// NewCmdStatus returns a new instance of a `rollouts status` command
+func NewCmdStatus(o *options.ArgoRolloutsOptions) *cobra.Command {
+	statusOptions := StatusOptions{
+		ArgoRolloutsOptions: *o,
+	}
+
+	var cmd = &cobra.Command{
+		Use:          "status ROLLOUT_NAME",
+		Short:        "Show the status of a rollout.",
```

Comment:
Argo isn't entirely consistent here, but generally doesn't include a `.`:
```suggestion
		Short:        "Show the status of a rollout",
```